package.json

```diff
@@ -1955,6 +1955,7 @@
         "stack-trace": "0.0.10",
         "string-argv": "^0.3.1",
         "strip-ansi": "^5.2.0",
+        "styled-components": "^5.2.1",
```

Comment:
These were required by @nteract/presentational-components which is required by the new vega version when building, but didn't seem to be picked up in the dependency add, so manually added here.